mirrord/cli/src/main.rs

```diff
@@ -263,8 +264,20 @@ fn main() -> Result<()> {
         Commands::Exec(args) => exec(&args)?,
         Commands::Extract { path } => {
             extract_library(Some(path))?;
-        } // Commands::Login(args) => login(args)?,
+        }
+        // Commands::Login(args) => login(args)?,
+        Commands::Operator(operator) => match operator.command {
+            OperatorCommand::Setup { file, namespace } => {
```

Comment:
Print out the explanation:
This will install the operator, which requires a seat based license to be used.
The features it enables are documented here:
You don't need to install the operator to use open source mirrord features.

Reply:
Should this even be available if the user isn't logged in?

Reply:
> Should this even be available if the user isn't logged in?

I think for the initial phase that's okay (until we build a backend for it..?)
If we can spin up something quite fast then maybe block it on login.

Reply:
let's spin up a simple backend, discuss it with me on Discord.